components/script/dom/bindings/codegen/Configuration.py

```diff
@@ -336,6 +342,10 @@ def isGlobal(self):
 
 
 # Some utility methods
+def getModuleFromObject(object):
+    return object.location.filename().split('/')[-1].split('.webidl')[0] + 'Binding'
```

Comment:
Which was written by somebody who only knows This-Does-Not-Raise-An-Exception-So-Ship-It Python (aka me).